lib/py-ios-device-client.js

```diff
@@ -62,6 +82,9 @@ class Pyidevice {
     }
   }
 
+  /**
+   * @return {Promise<object>}
```

Comment:
if we know what this object looks like, we should type it, but it doesn't need to block the PR